examples/nirfsa/getting-started-iq.py

```diff
@@ -60,78 +61,53 @@
 vi = None
 
 
-def raise_if_error(response):
-    """Raise an exception if an error was returned."""
-    if response.status != 0:
-        error_response = client.ErrorMessage(
-            nirfsa_types.ErrorMessageRequest(status_code=response.status)
-        )
-        raise Exception(f"Error: {error_response.error_message}")
-
-    return response
-
-
-def raise_if_initialization_error(response):
-    """Raise an exception if an error was returned from Initialize."""
-    if response.status < 0:
-        raise RuntimeError(f"Error: {response.error_message or response.status}")
+def check_for_warning(response, vi):
+    """Print to console if the status indicates a warning."""
     if response.status > 0:
-        sys.stderr.write(f"Warning: {response.error_message or response.status}\n")
-    return response
+        warning_message = client.ErrorMessage(
+            nirfsa_types.ErrorMessageRequest(vi=vi, error_code=response.status)
+        )
+        sys.stderr.write(f"{warning_message}\nWarning status: {response.status}\n")
```

Comment:
I know you copied this, but I think it's wrong in my original.

The data type of `warning_message` is not a string, it's `ErrorMessageResponse`. Which means what we print to the console should be `warning_message.error_message`. Right @reckenro ?

@danestull if your PR isn't submitted yet, can you include this fix in yours too?

Reply:
Oh yes, good catch. I missed this in that initial PR, it should be `warning_message.error_message` here like you said.